planner/core/plan_stats_test.go

```diff
@@ -183,6 +183,16 @@ func TestPlanStatsLoad(t *testing.T) {
 				require.Greater(t, countFullStats(reader.Stats().HistColl, tableInfo.Columns[2].ID), 0)
 			},
 		},
+		{ // check idx(b)
+			sql: "select * from t USE INDEX(idx) where b >= 10",
+			check: func(p plannercore.Plan, tableInfo *model.TableInfo) {
+				pr, ok := p.(*plannercore.PhysicalIndexLookUpReader)
+				require.True(t, ok)
+				pis, ok := pr.IndexPlans[0].(*plannercore.PhysicalIndexScan)
+				require.True(t, ok)
+				require.True(t, pis.Stats().HistColl.Indices[1].IsEssentialStatsLoaded())
```

Comment:
Can the case test sync load of index stats? Index stats are loaded by default and haven't been evicted here.

Reply:
Sure, the test will clear the statsCache for each test case, search for code:
```go
dom.StatsHandle().Clear() // clear statsCache
```